### PR TITLE
fillers/withdrawals/withdrawals: Update to use Gwei instead of Wei.

### PR DESCRIPTION
**Pull request by spencer-tb**
PR for #30.

### CODE REVIEW

File: fillers/withdrawals/withdrawals.py [hunk 1]
```diff
@@ -77,7 +79,7 @@ def test_withdrawals_use_value_in_tx(_):
         index=0,
         validator=0,
         address=TestAddress,
-        amount=tx.gas_price * tx.gas_limit + 1,
+        amount=(tx.gas_price * tx.gas_limit + ONE_GWEI),
```
Comment (marioevz) [inline fillers/withdrawals/withdrawals.py:82]: There is a problem when specifying `(tx.gas_price * tx.gas_limit)` here, because their unit is wei, but `withdrawal.amount` expects gwei, and this multiplication actually translates to `(tx.gas_price * tx.gas_limit * 1e9)` in post's balance, so the test is failing.

One trick could be to set the gas_price to `ONE_GWEI` and remove `tx.gas_price` from this formula.

Another thing to consider is that, when we use `ONE_GWEI` in the withdrawal amount, it translates to `ONE_GWEI * ONE_GWEI` in the post balance, so we need to use a `1` here and check for `ONE_GWEI` in post.

**Review by marioevz — CHANGES_REQUESTED** [review]

**Comment by marioevz** [comment]
I think some of the tests are failing because of the issue described in my review.

The up-to-date go-ethereum branch with gwei withdrawals is https://github.com/lightclient/go-ethereum/tree/withdrawals-timestamp, so I think we can use this branch to double check that the changes are correct.

File: fillers/withdrawals/withdrawals.py (hunk 1 above)
Comment (spencer-tb) [reply fillers/withdrawals/withdrawals.py]: Apologies for this, a simple error on my part, after fixing the errors I realised I misunderstood the conversions. I've updated the PR with the commits below

File: fillers/withdrawals/withdrawals.py [hunk 2]
```diff
@@ -597,7 +599,7 @@ def test_withdrawals_zero_amount(_):
         amount=1,
     )
     block.withdrawals.append(withdrawal_2)
-    post[to_address(0x200)].balance = 1
+    post[to_address(0x200)].balance = ONE_GWEI
     yield BlockchainTest(pre=pre, post=post, blocks=[block])
```
Comment (marioevz) [inline fillers/withdrawals/withdrawals.py:603]: Could we add a new test case that uses the maximum value for the amount field in the withdrawal? I.e. use 2**64 -1 in amount. It should pass without issues.

**Review by marioevz — COMMENTED** [review]
Two minor comments, but overall looks very good! :D

File: fillers/withdrawals/withdrawals.py (hunk 2 above)
Comment (spencer-tb) [reply fillers/withdrawals/withdrawals.py:611]: Done! I've added to the commit below

File: setup.cfg [hunk 3]
```diff
@@ -72,6 +72,7 @@ extend-ignore =
     D414    # Ignore: Section has no content
     D415    # Ignore: First line should end with a period, question mark, or exclamation point
     D416    # Ignore: Section name should end with a colon
+    SC200   # Ignore: Spelling error in name (e.g. variable, function, class)
```
[the remark this answers is not in the document]
Comment (spencer-tb) [reply setup.cfg:75]: Ahh perfect! I will do that :)

**Review by marioevz — APPROVED** [review]
